ksp_plugin_adapter/localization/zh-cn.cfg

```diff
@@ -213,7 +213,7 @@ Localization {
     #Principia_FlightPlan_AddManœuvre = 新建轨道机动
     #Principia_FlightPlan_StatusMessage_ChangeFlightPlan = 请更换轨道规划
     #Principia_FlightPlan_StatusMessage_DeadlineExceeded = 轨道规划长度超过天体计算长度
-    #Principia_FlightPlan_StatusMessage_Tickle = 反复延长并缩短规划长度
+    #Principia_FlightPlan_StatusMessage_Tickle = shortening the flight plan or waiting for a few moments
```

Comment:
```suggestion
    #Principia_FlightPlan_StatusMessage_Tickle = 缩短规划长度或稍等片刻
```

Reply:
Ah, thanks a lot, done.  I wouldn't have pinged you since you mentioned that you were busy.